astropy/visualization/wcsaxes/coordinate_helpers.py

```diff
@@ -363,17 +381,34 @@ def set_ticks_visible(self, visible):
         """
         self.ticks.set_visible(visible)
 
-    def set_ticklabel(self, **kwargs):
+    def set_ticklabel(self, color=None, size=None, pad=None,
+                      exclude_overlapping=None, **kwargs):
         """
         Set the visual properties for the tick labels.
 
         Parameters
         ----------
+        size : float, optional
+            The size of the ticks labels in points
+        color : str or tuple, optional
+            A valid Matplotlib color for the tick labels
+        pad : float, optional
+            Distance in points between tick and label.
+        exclude_overlapping : bool, optional
+            Whether to exclude tick labels that overlap over each other.
         kwargs
-            Keyword arguments are passed to :class:`matplotlib.text.Text`. These
-            can include keywords to set the ``color``, ``size``, ``weight``, and
-            other text properties.
+            Other keyword arguments are passed to :class:`matplotlib.text.Text`.
+            These can include keywords to set the ``color``, ``size``,
+            ``weight``, and other text properties.
```

Comment:
`color` and `size` are now explicit keywords now.  Do you want to mention them again in `kwargs` (i.e. they can't be specified twice)?

Reply:
I've removed them from kwargs - thanks!